monai/networks/blocks/squeeze_and_excitation.py

```diff
@@ -183,10 +189,12 @@ def __init__(
             spatial_dims=spatial_dims, in_channels=n_chns_3, r=r, acti_type_1=acti_type_1, acti_type_2=acti_type_2
         )
 
-        if in_channels != n_chns_3:  # in the case of residual chns and output chns doesn't match
+        self.project = project
+        if self.project is None and in_channels != n_chns_3:
             self.project = Conv[Conv.CONV, spatial_dims](in_channels, n_chns_3, kernel_size=1)
-        else:
-            self.project = None
+
+        relu_type: Type[nn.ReLU] = Act[Act.RELU]
```

Comment:
could you make this relu optional, and support different types such as PReLU?

Reply:
Sure, I've modified.